fuel-core-interfaces/src/model/block.rs

```diff
@@ -88,4 +112,57 @@ impl FuelBlock {
             transactions: self.transactions.iter().map(|tx| tx.id()).collect(),
         }
     }
+
+    /* TODO for functions bellow, they are mostly going to be removed
+    https://github.com/FuelLabs/fuel-core/issues/364 */
+
+    pub fn transaction_data_lenght(&self) -> usize {
+        self.transactions.len() * 100
+    }
+
+    pub fn transaction_data_hash(&self) -> Bytes32 {
+        Bytes32::zeroed()
+    }
+
+    pub fn validator_set_hash(&self) -> Bytes32 {
+        Bytes32::zeroed()
+    }
+
+    pub fn transaction_sum(&self) -> usize {
+        0
+    }
+
+    pub fn withdrawals(&self) -> Vec<(Address, Word, AssetId)> {
+        vec![]
+    }
+
+    pub fn withdrawals_root(&self) -> Bytes32 {
+        Bytes32::zeroed()
+    }
+}
+
+/// This structure is ceated as placeholder for future usage.
+/// It represent commitment for next child block
+#[derive(Clone, Debug, Default)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct FuelBlockConsensus {
+    /// required stake for next block
+    pub required_stake: u64,
+    /// Map of Validator consensus key and pair of stake and signature
+    pub validators: HashMap<Address, (ValidatorStake, Bytes32)>,
```

Comment:
in the other struct, we map to a stake+address, and here to bytes32. why diverge the types?

Reply:
Needs to be changed, good catch i missed that one